qemu/tests/block_resize.py

```diff
@@ -72,17 +72,29 @@ def get_md5_of_file(filename):
         ex_args = (mpoint, filename) if os_type == 'windows' else filename
         return session.cmd(md5_cmd % ex_args).split()[0]
 
-    def check_shrink_completion_event(session, vol_id):
+    def check_shrink_completion(vol_id, size):
         """
         Check the disk extend or shrink finished in windows guest.
-
-        :param session: Session object.
         :param vol_id: Drive letter.
+        :param size: shrink size.
         """
-        cmd = params.get("check_258_event")
-        status, output = session.cmd_status_output(cmd)
-        msg = params.get("event_msg") % vol_id
-        return status == 0 and msg in output
+        def _check_event_cmd(cmd):
+            status, output = session.cmd_status_output(cmd)
+            test.log.debug(output)
+            test.log.debug(msg)
```

Comment:
If we add the debug log, please add the prompt message instead of (output) and (msg).  e.g. the event message is %s, % msg.  Or we can remove line 83 and line 84.  Thanks.

Reply:
Update